doc/CMakeLists.txt

```diff
@@ -41,9 +41,9 @@ if(NOT DEFINED SPHINX_OUTPUT_DIR)
   set(SPHINX_OUTPUT_DIR_LATEX ${CMAKE_CURRENT_BINARY_DIR}/latex)
   set(SPHINX_OUTPUT_DIR_PDF ${CMAKE_CURRENT_BINARY_DIR}/pdf)
 else()
-  set(SPHINX_OUTPUT_DIR_HTML ${SPHINX_OUTPUT_DIR})
-  set(SPHINX_OUTPUT_DIR_LATEX ${SPHINX_OUTPUT_DIR})
-  set(SPHINX_OUTPUT_DIR_PDF ${SPHINX_OUTPUT_DIR})
+  set(SPHINX_OUTPUT_DIR_HTML ${SPHINX_OUTPUT_DIR}/html)
```

Comment:
This will break our downstream builds, since we need to be able to specify exactly where the HTML is placed because we use intersphinx after to combine with our own doc.

Reply:
I propose we instead let users specify SPHINX_OUTPUT_DIR_HTML directly

Reply:
I was wondering about this. I'll just replace this patch with a comment describing what is going on. Thanks for the explanation.